### PR TITLE
Updating workflow naming and commit hooks

### PR DESCRIPTION
**Pull request by rmbielby**
## Pull request overview

Couple of different housekeeping bits here:

1. The workflow tests don't currently get picked up properly by branch protections as they're both assigned the same name! I've changed that name in each of the workflow scripts and once they've run once, I'll add them to the branch protections (which will mean they'll be part of protections for this PR and all following ones).
2. I've added tidy_code into the commit hook script. Thought I'd done this already, but can't see it anywhere in the template, so assuming I must have just added it to some of the live dashboards, but not here.

## Pull request checklist

Please check if your PR fulfils the following:
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been reviewed and added / updated if needed (for bug fixes / features)
- [x] Tests have been run locally and are passing (`run_tests_locally()`)
- [x] Code is styled according to tidyverse styling (checked locally with `tidy_code()`)

## What is the current behaviour?

Branch protections don't prevent PRs being completed when the tests fail.

## What is the new behaviour?

The tidy code and shiny tests appear as options in the branch protections panel:
image

And the tests are now listed as required in the PR:
image

### CODE REVIEW

**Comment by cjrace** [comment]
Seems to trigger and work well:
image

image

Will quickly fix the thing lintr flagged now

File: global.R [hunk 1]
```diff
@@ -112,8 +112,10 @@
 choicesAreas <- dfAreas %>%
   filter(geographic_level == "National") %>%
   select(geographic_level, area_name = country_name) %>%
-  rbind(dfAreas %>% filter(geographic_level == "Regional") %>%
-    select(geographic_level, area_name = region_name)) %>%
+  rbind(
+    dfAreas %>% filter(geographic_level == "Regional") %>%
```
[the remark this answers is not in the document]
Comment (cjrace) [reply global.R]: should have addressed this in the `tidy up lintr warning commit` I've just pushed

**Review by chfoster — APPROVED** [review]

**Review by cjrace — APPROVED** [review]

**Comment by rmbielby** [comment]
> 
> Will quickly fix the thing lintr flagged now

Lintr's done that itself in the PR right? If you look in file changes, it shows up as a change.

**Comment by cjrace** [comment]
> > Will quickly fix the thing lintr flagged now
> 
> Lintr's done that itself in the PR right? If you look in file changes, it shows up as a change.

It hadn't done as far as I'd seen - I thought it was just highlighting it? Though I may have just duplicated that if not

**Comment by rmbielby** [comment]
> > > Will quickly fix the thing lintr flagged now
> > 
> > 
> > Lintr's done that itself in the PR right? If you look in file changes, it shows up as a change.
> 
> It hadn't done as far as I'd seen - I thought it was just highlighting it? Though I may have just duplicated that if not

Something to test some time!

**Pull request merged**